scripts/travis/wait_for_browser_provider.sh

```diff
@@ -9,6 +9,7 @@ while [ ! -f $BROWSER_PROVIDER_READY_FILE ]; do
   let "counter++"
   if [ $counter -gt 240 ]; then
     echo "Timed out after 2 minutes waiting for browser provider ready file"
+    ./print_logs.sh
```

Comment:
can you please add a comment about the before_script / after_script issue?

otherwise lgtm